test/absinthe/schema/sdl_render_test.exs

```diff
@@ -208,10 +208,6 @@ defmodule Absinthe.Schema.SdlRenderTest do
       on :field
     end
 
-    directive :foo_foo do
-      on :field
-    end
```

Comment:
I thought this was a type system directive definition, but we're defining those using a [prototype schema](https://hexdocs.pm/absinthe/Absinthe.Schema.Prototype.html). I'm not clear on what kind of directive definition this is.